packages/ai-bot/helpers.ts

```diff
@@ -170,7 +175,7 @@ export function getRelevantCards(
   let sortedCards = Array.from(relevantCards.values()).sort((a, b) => {
     return a.id.localeCompare(b.id);
   });
-  return sortedCards;
+  return [mostRecentlySharedCard, sortedCards];
```

Comment:
This was necessary to tweak the system prompt message. If you just send a list of cards to the ai-bot, the ai-bot loses context of the actively shared card, particularly, when the conversation is long. Hence, I resorted to establishing the last shared card.